modules/ROOT/pages/mobile.adoc

```diff
@@ -33,11 +33,11 @@ include::partial$misc/mobile_platform_compatibility.adoc[]
 
 Add the following `meta` tag to the `head` of pages using {productname} to ensure the mobile user interface functions as intended.
 
-```html+++<meta name="viewport" content="width=device-width, initial-scale=1">++++++</meta>+++
-
+```html
+<meta name="viewport" content="width=device-width, initial-scale=1"></meta>
 ```
 
-To set mobile specific settings, add the setting to the link:{baseurl}/configure/editor-appearance/#mobile[`mobile`] configuration, such as:
+To set mobile specific settings, add the setting to the xref:configure/editor-appearance.adoc#mobile[`mobile`] configuration, such as:
```

Comment:
This xref doesn't resolve as the `mobile` id doesn't exist. It looks like the id is missing on the `mobile.adoc` configuration page.

- [ ] Fix broken xref

Reply:
Done! Added missing id in partial/configure/mobile.adoc